pkgs/development/interpreters/python/wrapper.nix

```diff
@@ -20,12 +20,16 @@ let
       fi
       mkdir -p "$out/bin"
 
-      cd "${python}/bin"
-      for prg in *; do
-        rm -f "$out/bin/$prg"
-        makeWrapper "${python}/bin/$prg" "$out/bin/$prg" --set PYTHONHOME "$out"
+      for path in ${stdenv.lib.concatStringsSep " " paths}; do
+        if [ -d "$path/bin" ]; then
+          cd "$path/bin"
+          for prg in *; do
+            rm -f "$out/bin/$prg"
+            makeWrapper "$path/bin/$prg" "$out/bin/$prg" --set PYTHONHOME "$out"
+          done
+        fi
       done
-    '' + postBuild;
```

Comment:
Note that I removed this because it caused an infinite recursion. No idea why it was there in the first place.